lib/checks/keyboard/has-at-least-one-main.js

```diff
@@ -0,0 +1,3 @@
+var main = document.querySelector('main,div[role="main"]');
```

Comment:
The existing check does not return the data and does not implement an after, so this check is better and more complete.